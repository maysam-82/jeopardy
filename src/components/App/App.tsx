import React from 'react';
import { Route, Switch } from 'react-router-dom';
import Home from '../../pages/Home';
import Category from '../../pages/Category';
import Header from '../Header';

const App = () => {
	return (
		<div className="container">
			<Header />
			<Switch>
				<Route exact path="/" component={Home} />
				<Route path="/category" component={Category} />
			</Switch>
		</div>
	);
};

export default App;
