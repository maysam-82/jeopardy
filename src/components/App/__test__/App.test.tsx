import React from 'react';
import { shallow } from 'enzyme';
import App from '../App';

describe('App component', () => {
	const component = shallow(<App />);
	it('should render App component', () => {
		expect(component).toMatchSnapshot();
	});
});
