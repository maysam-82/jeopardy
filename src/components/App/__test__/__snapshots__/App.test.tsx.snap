// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render App component 1`] = `
<div
  className="container"
>
  <Header />
  <Switch>
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(Home)",
          "type": [Function],
        }
      }
      exact={true}
      path="/"
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(Category)",
          "type": [Function],
        }
      }
      path="/category"
    />
  </Switch>
</div>
`;
